lib/components/poi_category_search_bar.dart

```diff
@@ -15,22 +15,44 @@ class PoiCategorySearchBar extends StatefulWidget {
   _PoiCategorySearchBarState createState() => _PoiCategorySearchBarState();
 }
 
+// TODO Get full list of categories
 class _PoiCategorySearchBarState extends State<PoiCategorySearchBar> {
   List<String> categories = <String>[
-    'architecture',
-    'bar',
-    'basilica',
-    'cathedral',
-    'church',
-    'exhibition',
-    'gas station',
-    'lake, ' 'monuments',
-    'museum',
-    'park',
-    'river',
-    'romanic',
-    'school',
-    'zoo',
+    'Alpin hut',
```

Comment:
I can't find any route with these poi's. Perhaps the reason is they all start with a capital letter.

Additionally jakub is in charge for displaying pois on the map, which is why he also should update the list of POI's.
In more detail jakub has to select a color a icon and a localization for the poi, therefore he needs to know which POI's we use.

Reply:
imo caps look better when displayed. Will add .toLowerCase

Reply:
The inserted tags are the most frequently used ones according to taginfo (https://taginfo.openstreetmap.org/keys/tourism#values). Roman and I agreed on these on our last meeting. Roman is in charge of route calculations, displaying them is only the next step and imo should have a default for "unknown" ones.

Reply:
Yes but since the poi ticket jakub worked on, it was more important for him to know which to use so he can also include colors and images for them. So please don't modify them anymore.